Assets/Scripts/Interactables/AttachableMovingObject.cs

```diff
@@ -4,6 +4,7 @@
 using Managers;
 using Player;
 using UI;
+using UnityEditor;
```

Comment:
remove this import (not sure if it'll fail builds, but not taking any chances with anything that says Editor)